extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.css

```diff
@@ -68,26 +68,28 @@
   z-index: 2147483642;
 }
 
-.i-amphtml-lbg-mask,
-.i-amphtml-lbg-gallery {
```

Comment:
Ah sorry I didn't mean that the position properties always had to be together. You could have left `.i-amphtml-lbg-gallery` in this rule here and then overridden the `top` property in its own rule to avoid duplication.